figures/generate-figures.sh

```diff
@@ -182,6 +182,32 @@ Rscript --vanilla scripts/figS6-subtype-umap-panels.R
 # Panels E and F were previously generated with `scripts/fig5_figS6-immune-deconv-panels.R` in a Figure 5 step
 
 
+##### Figure S7: Transcriptomics results for only high tumor-purity samples -------------
+
+
+# Run script to generate TP53 and EXTEND score boxplots and associated legend
+Rscript --vanilla scripts/figS7-tp53-telomerase-tumor-purity-threshold.R
+
+
+# Copy panels for this plot (specific panel order is TBD):
+
+# TP53 scores across TP53 status
+cp ${analyses_dir}/tp53_nf1_score/results/tumor-purity-threshold/tp53-score-status_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
+
+# TP53 expression across TP53 status
+cp ${analyses_dir}/tp53_nf1_score/results/tumor-purity-threshold/tp53-fpkm-status_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
+
+# TP53 classifier ROC curve
+cp ${analyses_dir}/tp53_nf1_score/results/tumor-purity-threshold/tp53-roc_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
```

Comment:
TP53 shuffle is cutoff in this panel currently. Probably an upstream issue but noting it here.

Reply:
Fixed here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1678/commits/34b9f40d39885607c3af6616f23997555e9ad010
Copied over to `figures/pdfs/supp/figs7/panels/` here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1678/commits/937adfbc6ff391acbf87fc9e3bde69831c331165